tests/components/light/test_init.py

```diff
@@ -356,7 +365,48 @@ def test_light_profiles(self):
         _, data = ent1.last_call("turn_off")
 
         assert not light.is_on(self.hass, ent1.entity_id)
-        assert {} == data
+        assert {light.ATTR_TRANSITION: 0} == data
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5872960)